.github/workflows/sigma-validation.yml

```diff
@@ -35,14 +35,23 @@ jobs:
   validate-sigma-rules:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
       - name: Set up Python 3.10
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v5
         with:
           python-version: "3.10"
       - name: Install dependencies
-        run: pip install check-jsonschema
+        run: pip install requests==2.31.0 check-jsonschema==0.28.0
       - name: Validate Sigma rules
         run: tests/validate-sigma-schema/validate.sh
```

Comment:
I think we can get rid of the shell file and call everything directly from the CI?

Reply:
Addressed in https://github.com/SigmaHQ/sigma/pull/4724/commits/76e24cb78ef34de8f083e873f5fd88471010cfe4.